customize/render/convert.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+// Why is this different than the one in renderfunctions.php?

```

Comment:
Good question. All I can see is that one of them takes individual paths as input, while the other one takes a folder as input and takes all PNG files in that folder.